nav2_smac_planner/test/CMakeLists.txt

```diff
@@ -115,4 +115,10 @@ ament_add_gtest(test_lattice_node test_nodelattice.cpp)
 
 ament_target_dependencies(test_lattice_node ${dependencies})
 
-target_link_libraries(test_lattice_node ${library_name})
+target_link_libraries(test_lattice_node ${library_name} 
+  ${library_name}_lattice
+)
+# if(WIN32)
```

Comment:
Commented out code? Should it be here and enabled or removed?

Reply:
this needs to be removed, made the change